packages/create-redwood-app/template/package.json

```diff
@@ -16,7 +16,7 @@
   },
   "engines": {
     "node": ">=14.17 <=16.x",
-    "yarn": ">=1.15 <2"
+    "yarn": ">=3"
```

Comment:
@thedavidprice the idea is that new projects would use yarn 3 out of the box right? So it'd be ok to set the version that way?

Reply:
Yes, indeed. Technically we'll be backwards compatible, correct.

BUT we need to be careful here because of the way Yarn v3 is "config-installed" vs installed like Yarn v1.

E.g. To install Yarn v3, you 1) globally install Yarn v1 and then 2) config a path to use Yarn v3

This means our create-redwood-app process will likely be run in a directory using Yarn v1. So we need to ensure that when the script checks `yarn --version` it's within the newly created project directory. Otherwise it will output `yarn 1.x` and then fail the engine check.

Making sense?